frontend/src/metabase/components/YearPicker/YearPicker.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+import { useArgs } from "@storybook/client-api";
+import YearPicker from "./YearPicker";
+
+export default {
+  title: "Parameters/YearPicker",
```

Comment:
I'm not entirely sure the name should be prefixed with `Parameters` since a year picker could be also used anywhere else, if necessary. 
wdyt @daltojohnso?

Reply:
I guess the category might be... "widget" or "picker" or something. Parameter seems fine to me for now--we can always reorganize later.